### DISCUSSION TITLE
Navigation options.

### DISCUSSION THREAD

**Discussion by EffeDion** [pelican-dev/panel#46] [Ideas]
Add an option to choose for left side navigation or top bar navigation. Similar to Pterodactyl 0.7 vs 1.0

I feel like the community is still quite split of this even after all this time of 1.0 alot still prefer the old 0.7 layout. So it would be amazing if we had a simple built-in switch option for this without having to do it through theming.

**Comment by Boy132** [comment]
Since beta2 that's possible. Set `FILAMENT_TOP_NAVIGATION` in your .env file to `true`. (We might add a toggle for that later.)
For now this only applies to the new admin area. Once the client area will be reworked too it will work in both places.

![grafik](https://github.com/pelican-dev/panel/assets/8203120/5295ce6b-a832-4f4c-bf96-74681b5bc29d)
